### PR TITLE
Implementation of fetchBefore

### PR DESCRIPTION
**Pull request by montaro**

### CODE REVIEW

File: app/im/instalk/store/Persistance.scala [hunk 1]
```diff
@@ -12,11 +12,20 @@ import play.api.libs.json._
...
+  def syncRoom(room: RoomId, seqNr: Long): Iterable[JsObject]
+
   def getTopic(room: RoomId): Option[String]
+
   def dropRoomHistory(room: RoomId): Unit
-  def dropAllRooms(): Unit //Dangerous!
+
+  def dropAllRooms(): Unit
+
+  //Dangerous!
```
Comment (AhmedSoliman) [inline app/im/instalk/store/Persistance.scala:28]: why did you move the comment here?

Comment (montaro) [inline app/im/instalk/store/Persistance.scala:28]: Auto IDE format I guess
